playbooks/roles/node/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+nvm_dir: /Users/travis/.nvm
+nvm_version: v0.33.2
+node_versions:
+- '6'
```

Comment:
I think it may be worth considering bumping these versions up. Node is currently on v10.

Reply:
This seems like a very good idea, not sure who all the stakeholders are for a decision like this. If it's just us, let's bump.

Reply:
Sounds great, I bet this is going to be very appreciated by users.